.travis.yml

```diff
@@ -1,17 +1,12 @@
 language: python
-sudo: false
 python:
     - 2.7
-    - 3.4
     - 3.5
     - 3.6
+    - 3.7
+    - 3.8-dev
     - pypy
     - pypy3
-matrix:
-    include:
-        - python: "3.7"
-          dist: xenial
-          sudo: true
```

Comment:
Do you need a `dist: xenial` at the top, or is that already the default for all Travis repos that don't specify an explicit `dist:`?

Reply:
Not any more - it seems :-)